Makefile

```diff
@@ -121,6 +122,13 @@ tests-unit: ## runs all unit tests
 tests-contract: ## runs all forge tests
 	cd contracts/evm && forge test
 
+near-da-rpc-sys:
+	rm -rf relayer/libs && \
+	mkdir relayer/libs && \
+	cd submodules/rollup-data-availability && \
```

Comment:
I'm unable to run `make near-da-rpc-sys` because the `submodules` folder is missing on my machine. Should we add it as a Git submodule?

Reply:
Indeed, it seems like I got confused and thought that repo was already a submodule.